Alicerce/Sources/View/UIColor.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  UIColor.swift
+//  Alicerce
+//
+//  Created by Luís Portela on 17/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import UIKit
+
+public extension UIColor {
+    private static let divisor = CGFloat(255)
+
+    private typealias Components = (red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat)
+
+    public convenience init(hex: String) {
+        let hex = hex.trimmingCharacters(in: .whitespacesAndNewlines)
+            .replacingOccurrences(of: "#", with: "")
+
+        var hexValue: UInt32 = 0
+
+        guard Scanner(string: hex).scanHexInt32(&hexValue) else {
+            fatalError("😱 Cannot convert string into `UInt32`")
```

Comment:
😢 i know, but i don't like to throw here, i have to make those validations later in the code, which makes it "dirty".
i know, it seems more correct to throw, but i think we will not be changing these values in runtime and initialising `UIColor` with those values, at least not most of the times.

I like to have my colours like this:

```swift
extension UIColor {
    static var tabbarTintColor: UIColor { return UIColor(hex: "#e73a35") }
}
```

If i `throw`, i have to make it `return try! UIColor(hex: "#e73a35")` (because i know that this will not fail) 😝 

Honestly i agree with you, but i don't like to throw in this particular case 😢 but...

and looking into the Apple API, they don't throwable init for `UIColor`